### DISCUSSION TITLE
Openssl - 3: How can I generate legacy.a file along with legacy.so

### DISCUSSION THREAD

**Discussion by Pankul94** [openssl/openssl#22353] [Q&A]
Is there way to generate the archive file for the legacy provider in Openssl - 3.

I am trying to statically link legacy.a. I am aware that I can use `no-shared` and `no-module` configure options to have the legacy provider inbuilt but is there a way to have the shared option to generate the legacy.so and have the archive file generated as well?

Thanks
Pankul

**Comment by mattcaswell** [comment]
No. This isn't possible.
